2651.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @param {number} arrivalTime
+ * @param {number} delayedTime
+ * @return {number}
+ */
+var findDelayedArrivalTime = function (arrivalTime, delayedTime) {
+  const sum = arrivalTime + delayedTime;
+
+  return sum >= 24 ? sum - 24 : sum;
```

Comment:
```suggestion
  return sum % 24;
```
FYI you can also do this